src/app/stake/btc/page.tsx

```diff
@@ -124,75 +134,143 @@ const StakeBTCPage = () => {
       );
   }
 
+  // currently adding some dummy data to test the Delegations component
+
+  const globalParams: GlobalParamsVersion = {
+    version: 1,
+    activationHeight: 680000,
+    stakingCapSat: 5000000000,
+    stakingCapHeight: 680500,
+    tag: "v1.0",
+    covenantPks: ["pk1", "pk2", "pk3"],
+    covenantQuorum: 3,
+    unbondingTime: 7200,
+    unbondingFeeSat: 10000,
+    maxStakingAmountSat: 1000000000,
+    minStakingAmountSat: 100000,
+    maxStakingTimeBlocks: 10000,
+    minStakingTimeBlocks: 100,
+    confirmationDepth: 6,
+  };
+
+  const btcNetwork = {
+    messagePrefix: "\u0018Bitcoin Signed Message:\n",
+    bech32: "bc",
+    bip32: {
+      public: 76067358,
+      private: 76066276,
+    },
+    pubKeyHash: 0,
+    scriptHash: 5,
+    wif: 128,
+  };
+
+  const queryMeta = {
+    next: () => {
+      // logic to fetch next page
+    },
+    hasMore: true,
+    isFetchingMore: false,
+  };
+
   return (
-    <div className="lg:w-1/2 mx-auto px-4 md:px-16 lg:px-0">
-      <h1 className="text-xl font-bold mb-8 text-gray-700">
-        Select Finality Provider
-      </h1>
-      <div>
-        <ScrollArea.Root className="max-h-[80vh] rounded-3xl border overflow-auto bg-white">
-          <ScrollArea.Viewport className="size-full rounded">
-            <Table.Root size="2" className="relative">
```

Comment:
can you make sure not to undo the styling?

Reply:
Yeah, it looks like my branch isn't in sync with master. Hence the mismatch. Let me rebase and sync the styling